torch_xla/csrc/ops/device_data.cpp

```diff
@@ -13,16 +13,34 @@ DeviceData::DeviceData(std::shared_ptr<torch::lazy::BackendData> data)
               /*hash_seed=*/(uint32_t)101),
       data_(std::move(data)) {}
 
+DeviceData::DeviceData(std::shared_ptr<torch::lazy::BackendData> data,
+                       torch::lazy::OpList ops, xla::Shape xla_shape,
+                       xla::OpSharding sharding)
+    : XlaNode(xla_device_data, ops, {data->shape()}, xla_shape, sharding,
+              /*num_outputs=*/1,
+              /*hash_seed=*/(uint32_t)101),
+      data_(std::move(data)) {}
+
 std::string DeviceData::ToString() const {
   std::stringstream ss;
   ss << XlaNode::ToString() << ", device=" << data_->device();
   return ss.str();
 }
 
+torch::lazy::NodePtr DeviceData::Clone() const { return Clone({}); }
```

Comment:
Is this calling?
```
// TODO(https://github.com/pytorch/xla/issues/4567) Remove this clone method
  virtual torch::lazy::NodePtr Clone(torch::lazy::OpList operands) const;
  ```
  Let's add some comments why we are passing an empty `oplist` argument here.
  
  Will this be removed/updated when we address https://github.com/pytorch/xla/issues/4567?

Reply:
I see that you have a similar comment below, 
```
// TODO(steventk) Ideally, we are either passing the actual oplist, or we
  // don't pass ops at all and use another XlaNode constructor
  ```

Reply:
I'll add the comment to where we make the call too